enhancements/installer/aws-load-balancer-type.md

```diff
@@ -0,0 +1,288 @@
+---
+title: aws-load-balancer-type
+authors:
+  - "@miheer"
+reviewers:
+  - "@wking"  
+  - "@Miciah"
+  - "@frobware"
+  - "@sdodson"
+approvers:
+  - "@wking"  
```

Comment:
You probably want [an installer approver][1] here instead of me.

[1]: https://github.com/openshift/installer/blob/master/OWNERS_ALIASES